package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "graasp-insights",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "https://github.com/graasp/graasp-insights.git",
```

Comment:
Use ssh+https. There's an example in one of our repos.

Reply:
I checked graasp-api, -service-analytics, -app-moodle-analytics, -app-data-visualization, -app-timer, -research, and -desktop; several of them don't have a "repository" key, and the only one that's different is the graasp-desktop one. I used the one in there.